src/main/java/de/tum/in/www1/artemis/service/ExerciseService.java

```diff
@@ -233,6 +233,65 @@ public void cleanup(Long id, boolean deleteRepositories) throws IOException {
         }
     }
 
+    @Transactional(readOnly = true)
+    public java.io.File exportParticipations(Long id, List<String> studentIds) {
```

Comment:
- please add JavaDoc comments for all public methods
- What kind of id do we expect here? Would be better to call it participationId